zproject/prod_settings_template.py

```diff
@@ -340,6 +341,40 @@
 #
 # SOCIAL_AUTH_SUBDOMAIN = 'auth'
 
+########
+## Generic OpenID Connect.
+
+from .config import get_secret
+
+## Note: The OIDC IdP is configured in a dictionary format
+## to allow future support of multiple IdPs, but for now
+## only one is allowed.
+SOCIAL_AUTH_OIDC_ENABLED_IDPS = {
+    ## You can change the idp_name below to a preferred name, but it
+    ## doesn't affect anything user-facing.
+    "idp_name": {
```

Comment:
Do we want this to be hardcoded to `oidc`, so that the right backwards-compatibility thing will happen when we add support for multiple?

Reply:
Hmm, what backwards-compatibility behavior do you mean? I don't think this will really impact anything. The endpoint for new providers will most likely still be `/complete/oidc/` and `login/social/oidc` just with an extra param (like SAML)